bno055/bno055.c

```diff
@@ -0,0 +1,168 @@
+#include "bno055.h"
+#include <nrf_drv_twi.h>
+#include "nrf_delay.h"
+
+
+static nrf_drv_twi_t twi = NRF_DRV_TWI_INSTANCE(0);
+
+void InitI2C()
```

Comment:
It doesn't appear to cause any trouble to do init the driver multiple times. It would probably be better to do the I2C init outside both TPS and BNO and then pass the result as an argument to TPS and BNO init